test/ex_webrtc/peer_connection_test.exs

```diff
@@ -199,6 +199,19 @@ defmodule ExWebRTC.PeerConnectionTest do
 
   # API TESTS
 
+  test "controlling process" do
```

Comment:
Not sure if we want this test. I feel like it's pretty ugly :thinking:

Reply:
I mean, if you have written it already...